### DISCUSSION TITLE
Welcome to mindfulnotifier Discussions!

### DISCUSSION THREAD

**Discussion by kmac** [kmac/mindfulnotifier#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by schnudeldudel** [comment]
This App is awesome. Works flawless with Amazfit Tools to get reminders on Huami Amazfit BIP.

**Reply by kmac** [reply]
Hey, thanks for the feedback! Glad you like it. And I'm also using the BIP, though I switched to use Gadgebridge instead of Amazfit Tools. It's a great little device. And I find getting random reminders in an unintrusive way can be quite powerful.
